packages/elements/src/clock/index.ts

```diff
@@ -207,7 +231,7 @@ export class Clock extends ResponsiveElement {
     return this._tick;
   }
   public set tick (value: boolean) {
-    const newValue = !!value;
+    const newValue = value;
```

Comment:
You don't need to declare `newValue`

Reply:
Updated to  remove newValue and use coming value instead.